tests/e2e/util.go

```diff
@@ -2694,7 +2714,8 @@ func waitForCSIMigrationFeatureGatesToggleOnkublet(ctx context.Context, client c
 		}
 		found = false
 		for annotation, value := range csinode.Annotations {
-			if annotation == migratedPluginAnnotation && value == vcpProvisionerName {
+			framework.Logf("Annotation seen on CSI node - %s:%s", annotation, value)
+			if annotation == migratedPluginAnnotation && strings.Contains(value, vcpProvisionerName) {
```

Comment:
Same comment as above

Reply:
this will not loop through all the nodes